### ISSUE TITLE
Aggregations arguments tooltips for Elasticsearch 6.4

### ISSUE THREAD

**Issue by mindrones** [nestauk/dapsboard#54]
When using the aggregation form, it would be useful to have a little `?` icon beside each argument label that we can hover to read a brief description of that argument so that we can learn the aggregations API as we use the tool.

We need to collect these brief texts from the official documentation of Elasticsearch 6.4 and eventually add them to the type system to be used in the form.

It could be a div at the bottom of the Query form that updates based on the current focus or on the hovered label if there’s no focused input.

Here's a mockup made in devtools:

<img width="496" alt="Screenshot 2020-07-16 at 22 41 31" src="https://user-images.githubusercontent.com/1309648/87725890-fe704500-c7b5-11ea-9092-d3fa4ccbe51e.png">

We might also add a ? icon as a hover target but we always update the content in the same place.

Basically:
- if we focus a field, update with docs for that field,
- if no field has focus, we show a message
- if we hover a ? icon, we update the content based on the label we hover
